app/src/main/kotlin/com/bq/arch_example/domain/moves/MovesStore.kt

```diff
@@ -40,4 +47,20 @@ class MovesStore(private val movesController: MovesController) : Store<MovesStat
             )
         )
     }
+}
+
+object MovesModule {
+    fun create() = Kodein.Module("movesStore") {
+        bindStore { MovesStore(instance()) }
+        bind<MovesController>() with singleton {
+            MovesControllerImpl(
+                instance(),
+                instance()
```

Comment:
I would put it in the same line, there is no intereset in this, it is always going to be just `instance()`
`MovesControllerImpl(instance(), instance())`

Same in the other modules in other stores

Reply:
70e5a3c